src/dash-table/components/Table/Table.less

```diff
@@ -436,6 +437,10 @@
         background-color: rgb(250, 250, 250);
     }
 
+    .dash-spreadsheet-inner td {
+        background-color: white;
+    }
```

Comment:
Does this have a visual impact?

Reply:
No. A style was applied on `tr` and was removed which caused issues with fixed columns (columns content was visible as it was scrolled under the fixed ones). Adding it back on `td` instead.. just because that way it's on cells.. and since everything is otherwise applied on cells, ever so slightly cleaner. There's no real difference between applying that style on tr or td in the end.